practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,68 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
+import { getAllCustomerService } from "../../services/customerService";
+import { v4 as uuidv4 } from "uuid";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+
+  useEffect(() => {
+    const getAllCustomers = async () => {
+      // Call getAllCustomerService function to get all data
+      const data = await getAllCustomerService();
+      setCustomers(data);
+    };
+    getAllCustomers();
+  }, []);
+
+  const renderList = () => {
+    return (
+      <ul className="customer__list">
+        {customers.map((customer) => (
+          <li className="customer__item" key={uuidv4()}>
+            <CustomerItem
+              avatar={customer.avatar}
+              name={customer.name}
+              email={customer.mail}
+              phoneNumber={customer.phoneNumber}
+              gender={customer.gender}
+            />
+          </li>
+        ))}
+      </ul>
+    );
+  };
+
+  return (
+    <div className="analytics">
+      <div className="analytics__header">
+        <h1 className="analytics__title">Customer List</h1>
```

Comment:
why need to use the h1 in here?

Reply:
Because header tags are often used to break up the content to make it easier to read and make it easier for someone to find what they want.
`h1` tag that I used in here help for making highlight the title of the website

Reply:
Title of the website is not Customer List. If you active on Dashboard tab the tile of section is Product Analytics.